srcpkgs/walledin/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'walledin'
+pkgname=walledin
```

Comment:
Why not make the pkgname WalledIn

Reply:
the binary installed is called `walledin`

btw do you know what the arm gcc errors are?

Reply:
I think project name trumps binary name -- 

it doesn't look like it used the right gcc for the command. should've been the ${XBPS_CROSS_TRIPLET}-gcc